### PR TITLE
Create Mute command

### PR DESCRIPTION
**Pull request by TheSkele27**
# Discord Information #
Discord Username w/ Discriminator:
Discord User ID:

**What is this Pull Request for?**

**Are you fixing an issue, if so, what is it?**

**What changes are being made in this pull request?**

**Put any additional details below**

### CODE REVIEW

**Comment by Krunniee** [comment]
You pulled it off, now get a cookie 🍪

**Comment by TheSkele27** [comment]
> You pulled it off, now get a cookie 🍪

Thanks for the cookie <3 🍪

File: commands/mute.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  guildOnly: true,
+  aliases: [],
+  permLevel: 'Server Moderator'
+};
+  
+exports.help = {
+  name: 'mute',
+  category: 'Moderation',
+  description: 'Mutes a user for a certain amount of time.',
+  usage: 'mute @user <time>]'
```
Comment (CoalSephos) [inline commands/mute.js]: `<time>]'`
The `[` seems out of place here?

File: commands/mute.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+const ms = require(`ms`);
```
Comment (CoalSephos) [inline commands/mute.js]: Shouldn't be a template string, consider replacing with ' '

File: commands/mute.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+module.exports.run = async (bot, message, args) => {
+
+  const tomute = message.guild.member(message.mentions.users.first() || message.guild.members.get(args[0]));
+  if (!tomute) return message.reply(`Couldn't find user.`);
```
Comment (CoalSephos) [inline commands/mute.js]: Again, template string.

File: commands/mute.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  if (tomute.hasPermission(`MANAGE_MESSAGES`)) return message.reply(`Can't mute them!`);
+  let muterole = message.guild.roles.find(`name`, `Muted`);
```
Comment (CoalSephos) [inline commands/mute.js]: Template string, replace with ' ' on both occurrences.
(`name`, `Muted`);

File: commands/mute.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  //start of create role
+  if (!muterole) {
+    try {
+      muterole = await message.guild.createRole({
+        name: `Muted`,
```
Comment (CoalSephos) [inline commands/mute.js]: Template string.

File: commands/mute.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        color: `#818386`,
```
Comment (CoalSephos) [inline commands/mute.js]: Template string.

**Pull request closed without merging**